src/processor/map/map_copy_from.cpp

```diff
@@ -56,6 +56,21 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapCopyFrom(LogicalOperator* logic
     }
 }
 
+static std::pair<column_id_t, bool> getWarningData(const BoundCopyFromInfo* copyFromInfo) {
+    column_id_t numWarningDataColumns = 0;
+    bool ignoreErrors = CopyConstants::DEFAULT_IGNORE_ERRORS;
+    if (copyFromInfo->source->type == common::ScanSourceType::FILE) {
```

Comment:
Ditto let's remove this if check. Any ScanSource will output warningDataColumn, some might be 0 some may not.

Reply:
updated